saveSol.py

```diff
@@ -137,8 +137,8 @@ def genDesFile(sol, aircraft = 'D82', i = 0, swpt = False):
         lvt = sol('l_{vt}').to('m')
         crootvt = sol('c_{root_{vt}}').to('m')
         ctipvt = sol('c_{tip_{vt}}').to('m')
-        dxleadvt = sol('\\Delta x_{lead_{vt}}_Mission/ClimbSegment/ClimbP/AircraftP/VerticalTailPerformance')[0][0].to('m')
-        dxtrailvt = sol('\\Delta x_{trail_{vt}}_Mission/ClimbSegment/ClimbP/AircraftP/VerticalTailPerformance')[0][0].to('m')
+        dxleadvt = sol('\\Delta x_{lead_{vt}}').to('m')
+        dxtrailvt = sol('\\Delta x_{trail_{vt}}').to('m')
         tanvt = sol('\\tan(\Lambda_{vt})_Mission/Aircraft/VerticalTail/VerticalTailNoStruct')
```

Comment:
@bqpd I still feel like having to go `sol('\\tan(\Lambda_{vt})_Mission/Aircraft/VerticalTail/VerticalTailNoStruct')` instead of `sol('\\tan(\Lambda_{vt})')` is a limitation worth adresssing

Reply:
Do you have multiple variables called `\\tan(\Lambda_{vt})`? If you don't, the simpler access should work just fine.

Reply:
It's because of the relaxed constants heuristic. It makes ```sol('\\tan(\Lambda_{vt})')``` return both the variable and relax=1

Reply:
@bqpd is there a workarond?

Reply:
@bqpd bump.

Reply:
oh hmm. I could give the relaxvars a different name?